flow/layers/layer_tree.h

```diff
@@ -29,7 +29,8 @@ class LayerTree {
   //   layer tree performs any operations that require readback
   //   from the root surface.
   bool Preroll(CompositorContext::ScopedFrame& frame,
-               bool ignore_raster_cache = false);
+               bool ignore_raster_cache = false,
+               const SkRect& cull_rect = kGiantRect);
```

Comment:
Nit: I knew that Flutter used `kGiantRect` a lot but it seems to be fragile with local transforms with large scales. It might be more robust to create an `kInfiniteRect` and check it with `SkRect::isFinite()`.